ui/app/AppLayouts/Communities/views/CommunitySettingsView.qml

```diff
@@ -356,6 +356,8 @@ StatusSectionLayout {
             accounts: root.walletAccountsModel
             referenceAssetsBySymbolModel: root.tokensStore.assetsBySymbolModel
 
+            walletStore: RootStore
```

Comment:
We are trying to avoid using singleton stores. It would be nice if you pass it as a parameter instead (chain until _appMain_ if necessary)!